core/miner/storage_fsm/impl/sealing_impl.hpp

```diff
@@ -53,6 +54,7 @@ namespace fc::mining {
         std::shared_ptr<PreCommitPolicy> policy,
         std::shared_ptr<boost::asio::io_context> context,
         std::shared_ptr<Scheduler> scheduler,
+        std::shared_ptr<PreCommitBatcher> precommit_batcher_,
```

Comment:
```suggestion
        std::shared_ptr<PreCommitBatcher> precommit_batcher,
```

Reply:
fixed

Reply:
It's about naming class fields with trailing `_`. This is function argument, so no trailing `_`.